src/components/Button/ButtonContent/styled.ts

```diff
@@ -0,0 +1,54 @@
+import styled from 'styled-components'
+
+// @TODO: Solve how to change svg color
+export const ButtonIcon = styled.img`
+	width: 2.5em;
+`
+
+export const ButtonTitle = styled.span`
+	font-size: 1em;
+	text-transform: uppercase;
+	margin: 0.2em 0.5em;
+`
+
+export const ButtonSubtitle = styled.p`
+	font-size: 0.5em;
+	margin: 0;
+	padding: 0;
+`
+
+// @NOTE: The background color is better when it is transparent, but I don't know how to do it without rgba and opacity.
```

Comment:
Proč nemůžeme použít rgba a opacity? :thinking: Kvůli kompatibilitě se všemi/staršími prohlížeči? Asi zase diskuze na stand up.

Reply:
Opacity nemůžu použít, protože nechci, aby celá komponenta a její děti byly průhledné. Jinak řečeno chci průhledné pozadí nikoli text. Ohledně rgba, které běžně používám, tak netuším jak kombinovat s definovanými barvami?
A tohle mi neprošlo 😆😆: background-color: rgba(${({theme})=> theme.text....}, 0.3); 

Dívala jsem se ještě na možnosti s použitím position, které by způsobilo, že opacity bude pouze na rodiči nikoli dítěti, ale zase to dává jiná omezení, které mi nepřišli vhodné.

Reply:
Tak po několikátém pokusu o změnu mi až teď došlo, že kombinuji formát hex s rgb 🤔 Jelikož naše theme je zapsaná v hex formě, pokoušela jsem se najít řešení zde: https://stackoverflow.com/questions/23201134/transparent-argb-hex-value/28481374?fbclid=IwAR1SFe91TBZJVy_-Nxx6jC4i4ZO-BkoDhtWOlM000x6JgoshVul_oJtTVRc
Bohužel bez úspěchu. Transparentnost se nezměnila za to barva ano, tak nevím, kde je chyba. A tak mě napadlo, že bychom mohli naší nastavenou barevnou paletu převést do formátu rgb a to by mělo fungovat. Debatu o tom dáme asi na callu.